model-app.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Save the current directory
+echo $PWD
+source ../.env
+source ./env.sh #Contains: versions of software
+
+export SPACK_ROOT=/build-dir/ubuntu_xenial/spack
+source $SPACK_ROOT/share/spack/setup-env.sh
+
+# model_dependencies='spack dependencies ${model}@${model_version}'
+model_w_ver=${model^^}
+model_w_ver+="_VER"
+
+spack load ${model}@${!model_w_ver}
+spack find > dependencies-info.txt
+
+echo "Job is: $PBS_JOBID" 
+echo "Mom is: $(hostname)"
+echo "Running from $(pwd)"
+
+#NP=$((${AGAVE_JOB_NODE_COUNT}*${AGAVE_JOB_PROCESSORS_PER_NODE}))
+NP=$((${nx}*${ny}))
+set -x
+
+if test -f input.txt; then
```

Comment:
We should not assume the input file (1) is always named input.txt or (2) defined PX and PY as the number of procs. All these bits of data should probably be part of the json file.